core/modules/system/system.admin.inc

```diff
@@ -1607,10 +1607,14 @@ function system_run_cron_submit($form, &$form_state) {
  */
 function system_logging_settings($form, &$form_state) {
   $form['#config'] = 'system.core';
+
+  $config_error_level = config_get('system.core', 'error_level');
+  $settings_error_level = settings_get('error_level', $config_error_level);
+
   $form['error_level'] = array(
     '#type' => 'radios',
     '#title' => t('Error messages to display'),
-    '#default_value' => config_get('system.core', 'error_level'),
+    '#default_value' => $settings_error_level,
```

Comment:
We don't want to change the default here. It would create the same problem as in d7 where saving the form would overwrite the config with the settings.php setting.